src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/DisableFeatureFlag.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if !NETSTANDARD1_0
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.VisualStudio.TestPlatform.Utilities;
+
+/// <summary>
+/// !!!NEVER USE A FLAG TO ENABLE A FUNCTIONALITY!!!
+/// 
+/// The reasoning is:
+/// 
+/// * New version will automatically ship with the feature enabled. There is no action needed to be done just before release.
+/// * Anyone interested in the new feature will get it automatically by grabbing our preview.
+/// * Anyone who needs more time before using the new feature can disable it in the released package.
+/// * If we remove the flag from our code, we will do it after the feature is the new default, or after the functionality was completely removed from our codebase.
+/// * If there is a very outdated version of an assembly that for some reason loaded with the newest version of TP and it cannot find a feature flag, because we removed that feature flag in the meantime, it will just run with all it's newest features enabled.
+///
+/// Use constants so the feature name will be hosted directly inside the assembly that reference this one avoiding back compatibility issue.
+/// </summary>
+
+// !!! FEATURES MUST BE KEPT IN SYNC WITH https://github.com/dotnet/sdk/blob/main/src/Cli/dotnet/commands/dotnet-test/VSTestFeatureFlag.cs !!!
+internal partial class DisableFeatureFlag : IDisableFeatureFlag
+{
+    private static readonly Dictionary<string, bool> FeatureFlags = new();
+
+    // NEVER USE A FLAG TO ENABLE A FUNCTIONALITY
+    private const string VSTEST_FEATURE_DISABLE = nameof(VSTEST_FEATURE_DISABLE);
```

Comment:
That would make it `VSTEST_FEATURE_DISABLE_DISABLE_ARTIFACTS_POSTPROCESSING` if we use nameof. Can the prefix just by VSTEST_ ? We don't have to overcomplicate this, or overcommunicate that this disable is different from some other non-feature disable.

Reply:
also here, I wanted to enforce the disabling concept, but I agree the outcome is not great.